### DISCUSSION TITLE
Support Samsung Galaxy Store Publishing API

### DISCUSSION THREAD

**Discussion by PhilHannent** [fastlane/fastlane#18559] [Ideas]
Samsung have released an API for automating publishing to their store. It would be awesome of this was built into fastlane.

https://developer.samsung.com/galaxy-store/galaxy-store-developer-api/content-publish-api.html

**Comment by hamzaydia** [comment]
any update? a plugin?

**Comment by waterbang** [comment]
Samsung's content publishing API cannot be used and has been returning 302 redirection.
